docs/B-Computations/logic.md

```diff
@@ -24,7 +28,7 @@ This chapter introduces the selection and iteration constructs supported by the
 
 A structured program consists of sets of simple constructs, each of which has one entry point and one exit point.  Any programmer may replace one construct with an upgraded construct without affecting the other constructs in the program or introducing errors \("bugs"\). 
 
-![](https://ict.senecacollege.ca//~ipc144/pages/images/structured.png)
+![structured](../../static/img/image39.png)
```

Comment:
Suggestion to provide meaningful alt text. Example: `Diagram of a complete program consisting of multiple replaceable constructs`